starter/package.json

```diff
@@ -127,8 +127,8 @@
     "@angular/platform-server": "^8.2.0",
     "@angular/router": "^8.2.0",
     "@nationalbankbelgium/code-style": "^1.5.0",
-    "@nationalbankbelgium/stark-core": "latest",
-    "@nationalbankbelgium/stark-ui": "latest",
+    "@nationalbankbelgium/stark-core": "file:../dist/packages-dist/stark-core/nationalbankbelgium-stark-core-10.2.0-71462b2f.tgz",
```

Comment:
Shouldn't these changes be reverted? Normally the Starter should install the `latest` from all Stark packages right?